modules/navigation.js

```diff
@@ -0,0 +1,39 @@
+// Sections
+const list = document.querySelector('#List');
+const add = document.querySelector('#New');
+const contact = document.querySelector('#Contact');
+
+// Navigation links array
+const links = document.querySelectorAll('#myLinks a');
+
+add.classList.add('hidden');
+contact.classList.add('hidden');
+
+export default function InitializeNavigation() {
+  links.forEach((section) => {
+    section.addEventListener('click', (e) => {
```

Comment:
- [ ] Just as the previous reviewer noted, I would recommend you use the arrow function and not the obsolete function keyword, it is best practice in the industry to use the Arrow function instead of the function keyword so that the hoisting/runtime can be helped by the developer. Consider effect this change on all the functions.